### PR TITLE
ecma-376-fix

### PR DESCRIPTION
**Pull request by ImplFerris**
## What does this change?

Updated Paragraph property, run property, table(row,cell) property elements' order as per the ECMA-376 spec. 
 
## References

ECMA-376:
https://ecma-international.org/publications-and-standards/standards/ecma-376/

Related Issue:
https://github.com/bokuweb/docx-rs/issues/724

## Screenshots

If applicable, add screenshots to help explain your changes.
Validated the ooxml doc
ooxml-fix

## What can I check for bug fixes?

You can use the vscode ooxml validator (https://marketplace.visualstudio.com/items?itemName=mikeebowen.ooxml-validator-vscode) to check if generated docx is valid as per ooxml.

### CODE REVIEW

**Comment by bokuweb** [comment]
@ImplFerris Thanks for your great work!!!!!!!!!!!!

**Comment by bokuweb** [comment]
Thank you. Since the impact is significant, please give me some time.

File: docx-core/src/documents/elements/paragraph.rs [hunk 1]
```diff
@@ -296,27 +296,48 @@ impl Paragraph {
     }
 
     pub fn size(mut self, size: usize) -> Self {
-        self.property.run_property = self.property.run_property.size(size);
+        self.property.run_property = Some(
```
Comment (bokuweb) [inline docx-core/src/documents/elements/paragraph.rs:299]: 👍

File: docx-core/src/documents/elements/paragraph_property.rs [hunk 2]
```diff
@@ -286,17 +331,19 @@ mod tests {
...
-            r#"<w:pPr><w:rPr /><w:ind w:left="20" w:right="0" /></w:pPr>"#
+            r#"<w:pPr><w:ind w:left="20" w:right="0" /></w:pPr>"#
         );
     }
 
     #[test]
     fn test_keep_next() {
         let c = ParagraphProperty::new();
         let b = c.keep_next(true).build();
+        dbg!(&str::from_utf8(&b).unwrap());
```
Comment (bokuweb) [inline docx-core/src/documents/elements/paragraph_property.rs:342]: Please remove dbg!

**Comment by bokuweb** [comment]
@ImplFerris 

LGTM!!
for now I'll merge  into 0.5.0-dev branch to evaluation.
sorry, could you resolve conflict 🙏

**Comment by bokuweb** [comment]
[question] is there way to validate ecma spec in ci?

**Comment by ImplFerris** [comment]
> [question] is there way to validate ecma spec in ci?

 I'll need to look into, I'll get back to you on this

**Pull request merged**